### ISSUE TITLE
Image alt text popups not following theme

### ISSUE THREAD

**Issue by bremensaki** [nileane/TangerineUI-for-Mastodon#132]
The pop-up window with the alt text for an image is always displayed as light text on a dark background, even in light mode. Ideally this should be inverted when using light themes for readability.

![image](https://github.com/user-attachments/assets/6179c331-0151-406a-bafb-fee867f5c359)
![image](https://github.com/user-attachments/assets/39295c0f-b46c-4db0-bcf7-d9939b88530f)

**Comment by nileane** [comment]
Hi,

This is intended. My impression is that a white text on a dark background actually improves readability in most cases since this element is always overlayed over an image or a video.

An alternative would be to make this element opaque instead of semi-transparent, but I do think the translucency works well in this context and should be kept.

**Comment by bremensaki** [comment]
I use and require "light mode" because I experience halation of text in dark themes. I often need alt text, for example, when I encounter a screenshot from a dark themed app. In this case, the alt text display has the exact same issue as the source material that I need it for.

**Comment by nileane** [comment]
Done! https://github.com/nileane/TangerineUI-for-Mastodon/commit/2f898373d774a885e54d41f1a43cf9b9fd2ab264
